android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIProgressBar.java

```diff
@@ -141,4 +151,20 @@ protected void handleSetMessageColor(int color)
 	{
 		label.setTextColor(color);
 	}
+
+	protected void handleSetTintColor(int color)
+	{
+		ColorStateList singleColorStateList = ColorStateList.valueOf(color);
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Minor, it may as well go inside the `if`
```Java
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
	final ColorStateList singleColorStateList = ColorStateList.valueOf(color);
	progress.setProgressTintList(singleColorStateList);
}
```

Reply:
Nice catch!